configure.ac

```diff
@@ -317,6 +317,20 @@ AM_CONDITIONAL([MAINTAINER_TOOLS],[test x"${i_do_have_maintainer_tools}" = xyes]
 AC_SUBST([MAINTAINER_TOOLS],["${i_do_have_maintainer_tools}"])
 
 #--------------------------------------------------------------------------
+# Check for autotrace and potrace (don't do anything with this yet)
```

Comment:
I'm still not sure I see at as being useful. Did said user also compile from source? Otherwise this is a pretty pointless check, since they will never see that it indicated yes/no to autotrace/potrace.

I think this is an indicator that better documentation or other indicators within FontForge itself are necessary, but not to the build system itself. Doing so implies that it's a build time dependency, which it's clearly not. I can certainly see this itself becoming a point of confusion because now people can be wondering if they need to install autotrace/potrace for the build to work, or if it will influence the build in any way.

Furthermore, this particular check can do no more - it doesn't (and shouldn't) influence compilation in any particular way.